### ISSUE TITLE
Mass Obsoletion: focus on branch 'metabolic disease' (MONDO:0005066)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6750]
branch: 'metabolic disease' (MONDO:0005066)
Terms to obsolete: 

obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0019597 | 46,XY disorder of sex development due to isolated 17,20-lyase deficiency
http://purl.obolibrary.org/obo/MONDO_0100189 | apolipoprotein A-I deficiency
http://purl.obolibrary.org/obo/MONDO_0044655 | c12orf65-related combined oxidative phosphorylation defect
http://purl.obolibrary.org/obo/MONDO_0019213 | cerebral organic aciduria
http://purl.obolibrary.org/obo/MONDO_0019800 | chronic hepatic porphyria
http://purl.obolibrary.org/obo/MONDO_0018292 | congenital disorder of glycosylation-related bone disorder
http://purl.obolibrary.org/obo/MONDO_0018277 | congenital muscular dystrophy with cerebellar involvement
http://purl.obolibrary.org/obo/MONDO_0018279 | congenital muscular dystrophy without intellectual disability
http://purl.obolibrary.org/obo/MONDO_0018144 | congenital myasthenic syndromes with glycosylation defect
http://purl.obolibrary.org/obo/MONDO_0018318 | disorder of asparagine metabolism
http://purl.obolibrary.org/obo/MONDO_0018579 | disorder of ketone body transport
http://purl.obolibrary.org/obo/MONDO_0017709 | disorder of lipid absorption and transport
http://purl.obolibrary.org/obo/MONDO_0017745 | disorder of O-mannosylglycan synthesis
http://purl.obolibrary.org/obo/MONDO_0017743 | disorder of O-N-acetylgalactosaminylglycan synthesis
http://purl.obolibrary.org/obo/MONDO_0017744 | disorder of O-xylosyl/N-acetylgalactosaminylglycan synthesis
http://purl.obolibrary.org/obo/MONDO_0017742 | disorder of O-xylosylglycan synthesis
http://purl.obolibrary.org/obo/MONDO_0017760 | disorder of other vitamins and cofactors metabolism and transport
http://purl.obolibrary.org/obo/MONDO_0017756 | disorder of pterin metabolism
http://purl.obolibrary.org/obo/MONDO_0017132 | hereditary ATTR amyloidosis
http://purl.obolibrary.org/obo/MONDO_0019224 | inborn disorder of gamma-aminobutyric acid metabolism
http://purl.obolibrary.org/obo/MONDO_0019227 | inborn disorder of glycerol metabolism
http://purl.obolibrary.org/obo/MONDO_0016805 | isolated oxidative phosphorylation complex disorder
http://purl.obolibrary.org/obo/MONDO_0016815 | Leigh syndrome with leukodystrophy
http://purl.obolibrary.org/obo/MONDO_0016816 | Leigh syndrome with nephrotic syndrome
http://purl.obolibrary.org/obo/MONDO_0015902 | major hypertriglyceridemia
http://purl.obolibrary.org/obo/MONDO_0016794 | maternally-inherited mitochondrial myopathy
http://purl.obolibrary.org/obo/MONDO_0016797 | multiple mitochondrial DNA deletion syndrome
http://purl.obolibrary.org/obo/MONDO_0020275 | oculocutaneous or ocular albinism
http://purl.obolibrary.org/obo/MONDO_0016157 | qualitative or quantitative defects of fukutin
http://purl.obolibrary.org/obo/MONDO_0016183 | qualitative or quantitative defects of protein glycosyltransferase-like
http://purl.obolibrary.org/obo/MONDO_0016182 | qualitative or quantitative defects of protein O-mannose beta1, 2N-acetylglucosaminyltransferase
http://purl.obolibrary.org/obo/MONDO_0034217 | resistance to thyroid hormone due to a mutation in thyroid hormone receptor beta
http://purl.obolibrary.org/obo/MONDO_0019688 | sulfation-related bone disorder
http://purl.obolibrary.org/obo/MONDO_0020240 | syndromic retinitis pigmentosa
http://purl.obolibrary.org/obo/MONDO_0016803 | unspecified inborn mitochondrial disorder

**Comment by nicolevasilevsky** [comment]
not to self - I should cross reference the metabolic disease ticket when I do this one (#5680)

**Comment by nicolevasilevsky** [comment]
this is ready for review.

I cross-referenced all of these terms against terms in this [spreadsheet](https://docs.google.com/spreadsheets/d/1Bh8uw7TIwT5Nj0_ub7wRYmFWShIY52uuABjZNk8mZos/edit#gid=1081507745) from this ticket:
https://github.com/monarch-initiative/mondo/issues/4985

Note, some of the terms in the spreadsheet above do not exist in Mondo yet, so I classified them as 'metabolic disease' and we can add the more specific terms later.

**Comment by katiermullen** [comment]
@nicolevasilevsky I reviewed this spreadsheet. Let me know what you think!

**Comment by nicolevasilevsky** [comment]
Thanks @katiermullen! I made updates per your review, it is ready for ROBOT/mass obs/PR now

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1QRaZvmPBk0ifLKwvB6fHiEZpww-Prcp5/edit#gid=385348265


TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab)
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)
- [ ] remove parents in the ‘metabolic branch’ for terms in ("Reviewer-Leave Curator-Stay" Tab
- [ ] add new comments ("ROBOT Reviewer-Leave Curator-Stay" Tab)


Note: Once the to-do list is done, 9 terms for which reviewers were unsure will remain to be assessed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
